examples/layers/JSONLayers/IGN_MNT_HIGHRES.json

```diff
@@ -7,7 +7,7 @@
 	"updateStrategy": {
 		"type": 1,
 		"options": {
-			"groups": [3, 7, 11, 14]
+			"groups": [11, 14]
```

Comment:
Just  a little question: why `IGN_MNT_HIGHRES.json` is modified at all? Let's either keep it as it is if we plan on use it, or let's remove it entirely.